portfolio/src/main/webapp/projects.css

```diff
@@ -0,0 +1,69 @@
+.font {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.holder {
+    margin-bottom: 50px;
+}
+
+.navbar-link {
+    color: white;
+} :hover {
```

Comment:
You can put this on its own line. `navbar-link:hover` if you want to restrict to only elements with the class while hovered I think.

Reply:
fixed!